### PR TITLE
Invalid Consensus data assert occurs in 'combineCandidates()'

### LINKED ISSUE #1253: `combineCandidates()` should assert all the candidates are valid
This code is incorrect:

https://github.com/bpfkorea/agora/blob/d3b05dcea8414882180e5b73014f468c346974af/source/agora/consensus/protocol/Nominator.d#L685

This should not be a continue, but an `assert`. All of the candidates for the current slot height should by definition be valid.

### PR DESCRIPTION
**Pull request by TrustHenry**
Invalid candidates are being combined.

Fixes #1253

### CODE REVIEW

File: source/agora/consensus/protocol/Nominator.d [hunk 1]
```diff
@@ -686,10 +686,8 @@ extern(D):
                 auto data = deserializeFull!ConsensusData(candidate[]);
 
                 if (auto msg = this.ledger.validateConsensusData(data))
-                {
-                    log.error("combineCandidates(): Invalid consensus data: {}", msg);
-                    continue;
-                }
+                    assert(0, format!"combineCandidates: Invalid consensus data: %s"(
+                        msg));
                 else
```
Comment (AndrejMitrovic) [inline source/agora/consensus/protocol/Nominator.d]: Please turn this into:

```D
log.info("combineCandidates: {}", slot_idx);
```

So just remove the `else { }` part.

Comment (AndrejMitrovic) [reply source/agora/consensus/protocol/Nominator.d]: (unindent)

Comment (TrustHenry) [reply source/agora/consensus/protocol/Nominator.d]: Thanks, `else` removed.

**Comment by AndrejMitrovic** [comment]
Also could you just fix the commit message? it has a typo.

**Comment by TrustHenry** [comment]
> Also could you just fix the commit message? it has a typo.

I fixed it. and a style fix

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by AndrejMitrovic** [comment]
There is still a typo in the commit message. `combineCandids` => `combineCandidates`

**Comment by TrustHenry** [comment]
Oh! Thanks. I fixed a typo

**Review by AndrejMitrovic — APPROVED** [review]

**Pull request merged**